shared-bindings/gifio/OnDiskGif.c

```diff
@@ -69,6 +71,30 @@
 //|           # minus the overhead measured to advance between frames.
 //|           time.sleep(max(0, next_delay - overhead))
 //|           next_delay = odg.next_frame()
+//|
+//|     The displayio Group and TileGrid layers can be bypassed and the image can
+//|     be directly blitted to the full screen. This can give a speed-up of ~4x to
+//|     ~6x depending on the GIF and display. This requires an LCD that uses
+//|     standard codes to set the update area, and which accepts RGB565_SWAPPED
+//|     pixel data directly:
+//|
+//|     .. code-block:: Python
+//|       # Initial set-up the same as above
```

Comment:
Need a blank line to make sphinx happy, I think.
```suggestion
//|     .. code-block:: Python
//|
//|       # Initial set-up the same as above
```